probe/docker/container.go

```diff
@@ -206,6 +207,7 @@ func (c *container) GetNodeMetadata() report.NodeMetadata {
 	c.RLock()
 	defer c.RUnlock()
 
+	labels, _ := json.Marshal(c.container.Config.Labels)
```

Comment:
Uhm, even if they can't have *by construction* and you are checking that in unit tests?